styles/components/hero-section.css

```diff
@@ -0,0 +1,28 @@
+.hero__image {
+  border-radius: 20px;
+  display: block;
+  margin-bottom: 50px;
+}
+.hero {
+  margin-top: 76px;
+  text-align: center;
+}
```

Comment:
Please move the **B** of BEM to the top of this file.

Also: filename should be `hero` – like the classname (always).